eng/testing/linker/project.csproj.template

```diff
@@ -19,4 +21,33 @@
     {AdditionalProjectReferences}
   </ItemGroup>
 
+  <!-- Workaround when there is no vNext SDK available; copy known apphost/framework reference info from 8.0. -->
```

Comment:
Can we log an issue and point to it here, so that we update this when we get a 9.0 SDK?

Reply:
We should keep this permanently, it only does the update between when we update the TFM & when there's a vnext TFM available. We'll need it again when we update to `net10.0`

Reply:
Then let's move this out of the csproj.template and into the targets file:

https://github.com/dotnet/aspnetcore/blob/main/eng/testing/linker/SupportFiles/Directory.Build.targets

That is the more natural place. And then you don't even need to do any templating, you can just use `$(TargetFramework)` in the .targets file. (Actually you could here too.)

Reply:
Done